website/src/content/docs/docs/introduction/editor/vscode.md

```diff
@@ -35,3 +62,31 @@ tsp code uninstall
 # For VSCode insiders
 tsp code uninstall --insiders
 ```
+
+## Telemetry
+
+The extension for Visual Studio Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://privacy.microsoft.com/privacystatement) to learn more. This extension respects the `telemetry.telemetryLevel` setting which you can find more information in the [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).
+
+### What telemetry data are collected
+
+- OperationTelemetry
+
+| Telemetry name    | Type         | Example         |
+| ---------------    | ----------------- | ---------------- |
+| EventName          | string | "start-extension" for example |
+| ActivityId         | string |  |
+| StartTime          | datatime    |  |
+| EndTime            | datatime    |  |
+| Result             | string    | "success","fail","cancelled", etc. |
+| LastStep           | string    |  |
+
+- OperationDetailTelemetry
+
+| Telemetry name    | Type         | Example         |
+| ---------------    | ----------------- | ---------------- |
+| ActivityId          | string |  |
+| EmitterName         | string | "@typespec/http-server-csharp" for example |
+| EmitterVersion      | string |  |
+| CompilerVersion     | string |  |
+| CompilerLocation    | string | "global-compiler" for example |
```

Comment:
This also would be good to shw that you don't mean local path but just local vs global

Reply:
this is the purpose for example. but I can make it more clarified